crates/lovely-core/src/lib.rs

```diff
@@ -140,13 +174,15 @@ impl Lovely {
             start.elapsed().as_millis()
         );
 
-        Lovely {
+        let lovely = Lovely {
             mod_dir,
             is_vanilla,
             loadbuffer,
             patch_table,
             dump_all,
-        }
+        };
+        RUNTIME.set(lovely).unwrap_or_else(|_| panic!("Shit's erroring"));
```

Comment:
hahahahahahahhahahaha

Reply:
give actual message obv

Reply:
This should never panic since this code cannot be run until it is set by lovely and at that point lovely is set. I can add a proper error though